src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -8,7 +8,7 @@ Make sure you have completed the below steps:
 In your application directory, you should first install the necessary dependencies. Although Amplify In-App Messaging can be used as a standalone JavaScript library, the UI integration is currently available only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
 
 ```bash
-npm install -E aws-amplify@in-app-messaging aws-amplify-react-native@in-app-messaging amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker react-native-get-random-values react-native-url-polyfill
+npm install -E aws-amplify@in-app-messaging aws-amplify-react-native@in-app-messaging amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill
```

Comment:
Do we need `in-app-messaging` from both `aws-amplify@in-app-messaging` and `aws-amplify-react-native@in-app-messaging`?  Additionally, as we are planning to deprecate `aws-amplify-react-native` shouldn't this be removed at all?  @cshfang can you take a look at this?

Reply:
oh - I hadn't really touched that as a part of this PR - but I can remove the picker at least, and the in-app messaging docs will have to change when we have in-app going GA. What do you think @cshfang ?

Reply:
We can leave it for now. But if using `aws-amplify-react-native` for anything other than in-app-messaging, the picker might be needed.

Reply:
ok - let us leave it then. Anything else you need to approve this PR @chintannp ?